torch/testing/_internal/common_utils.py

```diff
@@ -1773,6 +1778,23 @@ def make_tensor(size, device: torch.device, dtype: torch.dtype, *, low=None, hig
         result = torch.repeat_interleave(result, 2, dim=-1)
         result = result[..., ::2]
 
+    if exclude_values:
+        if dtype in integral_types() or dtype is torch.bool:
+            replace_with = torch.tensor(1, device=device, dtype=dtype)
+        elif dtype in floating_types_and(torch.half, torch.bfloat16):
+            replace_with = torch.tensor(torch.finfo(dtype).eps, device=device, dtype=dtype)
```

Comment:
This is incorrect because it ignores the domain requested

The value should the (inclusive) left end point of the domain if that's real-valued, followed by eps, and there should be a check that eps is within the domain (the domain in this case would be [-inf, rhs) ). If eps > rhs, like if the domain is [-inf, -5) then an assertion can be triggered that the case is unsupported.

The documentation for how values are excluded will need an update, too

Reply:
I agree ont he point that we can go for `exclude_zero` for now and come look at this again when we feel there is a requirement.

Looking forward to hearing on this, should I implement `exclude_zero` in that case?

@kshitij12345 @mruberry

Reply:
I think `exclude_zero`  (justified by the op in this PR) should be good enough as we aren't sure about the use case for a general `exclude`.